src/components/shared/LrAdvanceSearch/LrAdvanceSearch.vue

```diff
@@ -141,15 +149,16 @@ import { LrFilterObject } from "../LrChips/LrFilterObject.model";
 export default class LrAdvanceSearch extends Vue {
   @Prop({ required: true }) private config: LrAdvanceSearchConfig;
 
-  private expand: boolean = false;
```

Comment:
not required to remove

Reply:
It is recommended to not type everything for which the type can be inferred.